ClinGen/clincoded
# Connects to #838 navigation to variant curation and gene curation

It's required in ticket #838 to add navigate links to starting pages of variant curation and gene curation respectively.

**Code Changes**
1. Added new objects for "New Variant Curation" link, "New Gene Curation" link and space in file `src/clincoded/static/components/app.js`.
2. Separated link and space with different style classes in file `src/clincoded/static/libs/bootstrap/navigation.js`.
3. Defined new style classes in file `src/clincoded/static/scss/clincoded/modules/_navbar.scss`.

**Test**
* After login, expect to see new links in header as:
![screen shot 2016-08-11 at 4 52 40 pm](https://cloud.githubusercontent.com/assets/9206696/17608819/0e9dd368-5fe4-11e6-8ebd-3f4c4fd847ee.png)
* Click "New Variant Curation", expect to go to page:
![screen shot 2016-08-11 at 4 53 47 pm](https://cloud.githubusercontent.com/assets/9206696/17608830/345daccc-5fe4-11e6-9e55-0848b8d67607.png)
* Click "New Gene Curation", expect to go to:
![screen shot 2016-08-11 at 4 54 46 pm](https://cloud.githubusercontent.com/assets/9206696/17608852/57792d08-5fe4-11e6-9674-c4d217ddcb6e.png)
* Click Logout and expect not to see new links.
![screen shot 2016-08-11 at 4 55 55 pm](https://cloud.githubusercontent.com/assets/9206696/17608872/823fbc00-5fe4-11e6-8274-ab6728a85085.png)
**End of Test**
